build fail on MS Windows

build fail on MS Windows
```
.\main.go:282:14: undefined: pty.Start
.\main.go:290:31: not enough arguments in call to syscall.Syscall
.\main.go:290:32: undefined: syscall.SYS_IOCTL
.\main.go:290:67: undefined: syscall.TIOCSWINSZ

Compilation finished with exit code 2
```
`go version go1.9.2 windows/amd64`, win 10 Pro

Hi @Raerten 

I fixed your first problem (PTY) in this branch: https://github.com/moul/sshportal/compare/dev/moul/support-windows?expand=1

However, I now have a new building problem that seems to be already known by the community: https://github.com/mattn/go-sqlite3/issues/27

```console
$ GOOS=windows go build -v -i -o tmp .
github.com/moul/sshportal/vendor/github.com/mattn/go-sqlite3
# github.com/moul/sshportal/vendor/github.com/mattn/go-sqlite3
../../../go/src/github.com/moul/sshportal/vendor/github.com/mattn/go-sqlite3/sqlite3_go18.go:18:10: undefined: SQLiteConn
```

---

Is this an option for you to run sshportal in Linux, for instance using a VM or just "Docker for Windows" which supports Linux binary very nicely

--- a/hidden.go
+++ b/hidden.go
@@ -1,3 +1,5 @@
+// +build !windows
+
 package main
 
 import (

